inductiva/coastal/bathymetry.py

```diff
@@ -363,13 +363,14 @@ def plot(
               defined.
         """
 
-        logging.info("Plotting the bathymetry on a uniform grid...")
-        depths_grid, _ = self._interpolate_to_uniform_grid(
-            x_resolution,
-            y_resolution,
-            threshold_distance=threshold_distance,
-            nullable=True,
-        )
+        if not self.is_uniform_grid():
+            logging.info("Plotting the bathymetry on a uniform grid...")
```

Comment:
I agree with this, but I am just thinking if we should keep the x_resolution, y_resolution at all or some better default values...

Moreover, can you comment if it is taking to much time to plot?

Reply:
Is plotting quite quickly, it doesnt take a lot of time

Reply:
Nice!